google-site-kit.php

```diff
@@ -45,7 +47,15 @@ function googlesitekit_activate_plugin( $network_wide ) {
 	if ( version_compare( PHP_VERSION, GOOGLESITEKIT_PHP_MINIMUM, '<' ) ) {
 		wp_die(
 			/* translators: 1: version number */
-			esc_html( sprintf( __( 'Site Kit requires PHP version %s', 'google-site-kit' ), GOOGLESITEKIT_PHP_MINIMUM ) ),
+			esc_html( sprintf( __( 'Site Kit requires PHP version %s or higher', 'google-site-kit' ), GOOGLESITEKIT_PHP_MINIMUM ) ),
+			esc_html__( 'Error Activating', 'google-site-kit' )
+		);
+	}
+
+	if ( version_compare( GOOGLESITEKIT_WORDPRESS_VERSION, GOOGLESITEKIT_WP_MINIMUM, '<' ) ) {
+		wp_die(
+			/* translators: 1: version number */
+			esc_html( sprintf( __( 'Site Kit requires WP version %s or higher', 'google-site-kit' ), GOOGLESITEKIT_WP_MINIMUM ) ),
```

Comment:
Sorry, seeing this in context now, I realize this should really say "WordPress" instead of the abbreviation. I've updated the AC 👍 

```suggestion
			esc_html( sprintf( __( 'Site Kit requires WordPress version %s or higher', 'google-site-kit' ), GOOGLESITEKIT_WP_MINIMUM ) ),
```